src/components/organisms/GlobalNav/GlobalNav.js

```diff
@@ -71,6 +72,19 @@ const GlobalNav = forwardRef((props, globalNavRef) => {
       .scrollIntoView({ behavior: "smooth", block: "start" });
   };
 
+  useEffect(() => {
+    const historyListener = globalHistory.listen(() => {
+      // detect navigation with browser arrows
+      if (!globalHistory.location.action) {
+        // reload the page to get it in proper position
+        window.location.reload();
```

Comment:
Seemed like the quickest, cleanest, nicest fix - just reloading the page so it finds its proper position on load. What do you think @willmcl